torch/distributed/optim/post_localSGD_optimizer.py

```diff
@@ -65,10 +66,19 @@ def __repr__(self):
         return self.optim.__repr__()
 
     def state_dict(self):
-        return self.optim.state_dict()
+        optim_state_dict = self.optim.state_dict()
+        if optim_state_dict is not None and self.averager.step is not None:
+            optim_state_dict['step'] = self.averager.step
+        return optim_state_dict
 
     def load_state_dict(self, state_dict):
         self.optim.load_state_dict(state_dict)
+        if 'step' in state_dict.keys():
```

Comment:
similar to above `state_dict` suggestion, can add a small docstring to `load_state_dict`